modules/installation-vsphere-infrastructure.adoc

```diff
@@ -24,3 +24,9 @@ clusters are subject to the following restrictions:
 
 If you use a vSphere version 6.5 instance, consider upgrading to 6.7U2 before
 you install {product-title}.
+
+[IMPORTANT]
+====
+Time configuration of ESXi hosts is critical for a successful installation of
```

Comment:
Do the hosts just need to be in sync, or is it better to enable the NTP client? I might say "You must ensure that the time on your ESXi hosts is synchronized before you install {product-title}."

Reply:
@kalexand-rh I think what you wrote sounds good.  I will update.
In regards to your other question I think NTP should always be enabled but I am sure there is some enivornment where that is not possible and will need to manually set.  As long as the time between the ESXi hosts and the machine that is running `openshift-installer` are in sync there shouldn't be a problem.